platforms/core-configuration/kotlin-dsl-tooling-builders/src/main/kotlin/org/gradle/kotlin/dsl/tooling/builders/internal/IsolatedProjectsSafeKotlinDslScriptsModelBuilder.kt

```diff
@@ -294,9 +294,11 @@ fun isolatedScriptsModelFor(project: ProjectInternal): IsolatedScriptsModel {
     // TODO:isolated compute own classpaths
     val additionalClassPath = ClassPath.EMPTY
     val additionalSourcePath = ClassPath.EMPTY
-    val models = buildList {
-        addNotNull(buildScriptModelFor(project, additionalClassPath, additionalSourcePath))
-        this.addAll(precompiledScriptModelsFor(project))
+    val models = mutableListOf<IntermediateScriptModel>().apply {
```

Comment:
Build manually until https://github.com/gradle/gradle/issues/28325 is fixed.

I tried using Guava, but it turns out this module does not have a direct dependency on it yet, so I'd avoid adding it just for this.